jsonrpcserver/async_dispatcher.py

```diff
@@ -100,5 +103,5 @@ async def dispatch_to_response_pure(
             )
         )
     except Exception as exc:
-        logging.exception(exc)
+        logger.error(traceback.format_exc())
```

Comment:
Would logger.exception with exc_info and/or stack_info params do what we need here?

Reply:
From my testing, the format with logger.exception with exc_info produces redundant information, hence using logger.error(traceback.format_exc()). This will keep the same format you would expect from an error on the console.

Reply:
I get the same output with and without your change

```
>>> import logging
>>> logger = logging.getLogger(__name__)
>>> try:
...  raise ValueError
... except ValueError as exc:
...  logger.exception(exc)
...

Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError
>>> import traceback
>>> try:
...  raise ValueError
... except ValueError:
...  logger.error(traceback.format_exc())
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError

```

Reply:
My mistake, thanks for confirming. Reverted.

Reply:
Just missed the same in async_dispatcher.py

Reply:
Apologies. Updated.